docs/Introduction.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 1
+sidebar_label: "♦️ What is RedStone Oracles?"
+---
+
+## Overview
+
+There is a growing need for decentralized applications (dApps) to access data feeds that are frequently updated, reliable, and secure. RedStone Oracles is a leading oracle provider directly fulfilling this need. Trusted by 100+ dApps and securing billions of dollars of value, RedStone provides customizable and cost-efficient data streams for builders empowering the next generation of dApps. RedStone also provides data feeds to blockchains and layer 2 scaling solutions across the entire blockchain ecosystem that are both EVM and non-EVM compatible. The current model of oracle systems suffers from key inefficiencies, all of which RedStone Oracles was specifically designed to solve from the bottom up. This makes RedStone a unique oracle service.
+
+## The Problems RedStone Solves ♦️
+
+### Problem # 1: The standard approach of providing data to applications is inefficient and expensive. 
+- The standard method of providing data is to ‘push’ the data onto the blockchain regardless of whether it is used by an application. This results in paying more for data and dedicating resources that could be used elsewhere. 
+
+#### Our Solution
+
+RedStone allows data to be provided on-demand rather than on a fixed schedule, reducing the costs of putting data 'on-chain'. This is achieved by storing data off of the blockchain as cryptography signed packages and allowing smart contracts of dApps to fetch data when necessary. 
+
+_A note on data integrity_: To maintain data integrity RedStone also provides data feeds to Arweave, a decentralized network that provides data storage. Arweave's decentralized network makes it tamper-proof, therefore any data provided by RedStone to Arweave acts as a unbiased source of truth about the historical data provided by RedStone. 
+
+Overall, RedStone's approach improves efficiency for dApps and significantly reduces the costs for dApps to access data feeds. 
+
+
+### Problem #2: The typical monolithic architecture of oracles limits scalability.
+
+- A consequence of a monolithic architecture is that makes it challenging for protocols to reduce latency and list new assets.
+
+#### Our Solution
+
+RedStone was designed with a modular architecture making it easy to incorporate new assets and reduce latency, allowing dApps to scale more efficiently.
+
+
+### Problem #3: dApps are overly dependent on relayers.
```

Comment:
I am not fully convinced to this point. We also use relayers in classic model and other oracles have separated parts with data fetching and pushing on-chain

Reply:
Is there a different solution you think I should include instead? If not, I will leave it at 2 solution.

Reply:
I cannot come up with something but you can ask someone from biz/marketing team as they could have something prepared